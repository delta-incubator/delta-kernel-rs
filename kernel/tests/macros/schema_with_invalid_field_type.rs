extern crate delta_kernel_derive;
use delta_kernel_derive::Schema;
use syn::Token;

mod actions;
mod schema;

#[derive(Schema)]
pub struct WithMacro {
    token: Token![struct],
}

fn main() {}
