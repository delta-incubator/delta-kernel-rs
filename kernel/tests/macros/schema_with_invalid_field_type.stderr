error: Can't handle type: Type::Macro { mac: Macro { path: Path { leading_colon: None, segments: [PathSegment { ident: Ident { ident: "Token", span: #0 bytes(162..167) }, arguments: PathArguments::None }] }, bang_token: Not, delimiter: MacroDelimiter::Bracket(Bracket), tokens: TokenStream [Ident { ident: "struct", span: #0 bytes(169..175) }] } }
  --> tests/macros/schema_with_invalid_field_type.rs:10:5
   |
10 |     token: Token![struct],
   |     ^^^^^
